src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/services/MailService.java

```diff
@@ -64,8 +76,8 @@ public boolean sendPassword(String to, String strongPassword) {
                     + "old_password=" + strongPassword;
 
             String htmlText = "<img src='cid:logo' alt='logo' height='200'> " +
```

Comment:
Du setzt in den anderen Methode extra den Text, aber verwendest ihn hier nicht. Im Text hier: DIDentity => zweites d bitte klein

Reply:
fixed